tests/test_cell_metadata.py

```diff
@@ -1,8 +1,16 @@
 import sys
 import unittest
+import json
+
+from mock_data.annotation.metadata.convention.valid_array_v2_1_2 import (
```

Comment:
How come this is done via an import?  Is it because `valid_array_v2_1_2.py` has `ObjectID()` values?  Couldn't we just have static values for those?